lib/ba/ba_support/reaction.rb

```diff
@@ -35,7 +35,11 @@ def self.base_reactions
           nya: Reaction.new('nya:434511854505558019', /\b(nya+[hn]?|me+ow)\b/),
           rat: Reaction.new('🐀', /\bsque+a+k\b/),
           train: Reaction.new('🚄', /\bchoo+\s*choo+\b/),
-          wooloo: Reaction.new('a:wooloo:605781427698663424', /\bwoo+loo+\b/)
+          wooloo: Reaction.new('a:wooloo:605781427698663424', /\bwoo+loo+\b/),
+          moon: Reaction.new(
+            'goodnight:600686474320412702',
```

Comment:
Where is this emoji coming from? I want al emojis to come from my dev server so that I can guarantee the bot always has access to it